### PR TITLE
VZ-1315 zap structured logging

### PR DESCRIPTION
**Pull request by guoyongzhang**

### CODE REVIEW

File: THIRD_PARTY_LICENSES.txt [hunk 1]
```diff
@@ -9662,6 +9648,17 @@ github.com/rs/cors
 -------- Copyrights
 Copyright (c) 2014 Olivier Poitrey <rs@dailymotion.com>
 
+-------- Dependency
+github.com/rs/xid
```
Comment (markxnelson) [inline THIRD_PARTY_LICENSES.txt:9652]: unapproved dep

File: cmd/manager/main.go [hunk 2]
```diff
@@ -176,3 +161,25 @@ func serveCRMetrics(cfg *rest.Config) error {
 	}
 	return nil
 }
+
+// Initialize logs with Time and Global Level of Logs set at Info
+func InitLogs() {
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+
+	// Log levels are outlined as follows:
+	// Panic: 5
```
Comment (markxnelson) [inline cmd/manager/main.go:170]: enum

**Review by markxnelson — DISMISSED** [review]

File: THIRD_PARTY_LICENSES.txt (hunk 1 above)
Comment (shihchang) [reply THIRD_PARTY_LICENSES.txt]: updated with latest attribution-helper

File: cmd/manager/main.go (hunk 2 above)
Comment (shihchang) [reply cmd/manager/main.go]: replaced to zap

File: pkg/controller/helidonapp/helidonapp_controller.go [hunk 3]
```diff
@@ -80,8 +77,7 @@ type ReconcileHelidonApp struct {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileHelidonApp) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	reqLogger.Info("Reconciling HelidonApp")
+	zap.S().Infow("Reconciling HelidonApp")
```
Comment (bobdonat) [inline pkg/controller/helidonapp/helidonapp_controller.go:80]: Looks like the Request.Namespace and Request.Name fields are not being logged with the new changes.  Without these fields there is no context as to what resource the controller is operating on.

Comment (shihchang) [reply pkg/controller/helidonapp/helidonapp_controller.go]: fixed

Comment (shihchang) [reply pkg/controller/helidonapp/helidonapp_controller.go]: `reqLogger := zap.S().With("Request.Namespace", request.Namespace, "Request.Name", request.Name)`

**Comment by bobdonat** [comment]
Curious why THIRD_PARTY_LICENSES.txt has so many diffs when there are no changes in go.mod/go.sum

**Comment by shihchang** [comment]
> Curious why THIRD_PARTY_LICENSES.txt has so many diffs when there are no changes in go.mod/go.sum

@bobdonat There is some changes in go.mod. attribution-helper has a recent update (from Mike) that removes lots unnecessary inclusions according to Paul.

**Review by bobdonat — APPROVED** [review]

**Pull request merged**